manuscript/draft.Rmd

```diff
@@ -507,219 +507,198 @@ fit.](../figures/observers.png)
-considering the potential importance of different models and processes for
-making forecasts at different scales. Our results show that for forecasts 1-3
-years in the future, baselines and time-series based approaches outperform all
-other methods using integrative metrics. At these time scales, environmental
-changes are relatively small and changes in the biota may lag behind changes in
-the environment. However, as the timescale of the forecast increases to a decade
-the JSDMs and environmental richness models get closer to the accuracy of the
-time-series methods and even surpass some of those methods. As the forecast
-distance increases the amount of environmental change is expected to increase
-and the system will have more time to respond, potentially leading to shifts
-towards improved relative performance of the models incorporating environmental
-and ecological information. If this shift with forecast distance in the relative
-importance of different processes and models continues, this could lead to JSDMs
-and environmental richness models outperforming time-series based approaches at
-sufficiently long time scales. These results suggest that assessment of how
-forecast performance changes with time lag is not only important for determining
-how far in the future to forecast, but also for determining which models and
-processes are most relevant for making forecasts at different time scales. For
-example ensemble forecasts could be created which incorporate these time lags
-into the weights for individual models.
-
-It is also possible that models that include species' relationships to their
-environments or direct environmental constraints on richness will continue to
-produce forecasts that are worse than simply assuming the systems are static.
-This would be expected to occur if richness in these systems is not changing
-over the relevant time scales (e.g. 10-30 years in this study), which would make
-simpler models with no directional change more appropriate. Recent suggestions
-that local scale richness in some systems is not changing directionally at
+The accuracy for forecasts is expected to decline as forecasts are made further
+into the future, so it is important to evaluate forecasts at different lead
+times [Box 1; @petchey2015]. For deviance (the error metric that evaluates both
+models' point estimates and their uncertainty simultaneously), there is some
+indication that the JSDM and the nonlinear richness model converge towards the
+time-series and baseline models as the lead time of the forecast increases. This
+potential convergence is biologically plausible because at short lead times,
+environmental changes are relatively small and changes in the biota may lag
+behind changes in the environment. However, as the lead time of the forecast
+increases the amount of environmental change should increase and the systems
+will have more time to respond, potentially resulting in shifts towards improved
```

Comment:
We should probably cite a recent Blonder paper here. It goes into all the details of the lag hypothesis. 

```
@article{blonder2017,
author = {Blonder, Benjamin and Moulton, Derek E. and Blois, Jessica and Enquist, Brian J. and Graae, Bente J. and Macias-Fauria, Marc and McGill, Brian and Nogu{\'{e}}, Sandra and Ordonez, Alejandro and Sandel, Brody and Svenning, Jens-Christian},
doi = {10.1111/ele.12736},
journal = {Ecology Letters},
month = {mar},
number = {3},
pages = {293--306},
title = {{Predictability in community dynamics}},
url = {http://doi.wiley.com/10.1111/ele.12736},
volume = {20},
year = {2017}
}
```

Reply:
Thanks. Done.

Reply:
Also added the Blois citation.